pkgs/development/python-modules/chromaprint/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, fetchFromGitHub
+, python3
+}:
+
+let
+  pname = "chromaprint";
```

Comment:
you can move pname and version into the buildPythonPackage call as it's being called with a recursive set anyway.